test/apps/export-webpack/test.ts

```diff
@@ -0,0 +1,19 @@
+import * as assert from 'assert';
+import * as api from '../../../api';
+import * as fs from 'fs';
+
+describe('export-webpack', function() {
+	this.timeout(10000);
+
+	// hooks
+	before(async () => {
+		await api.build({ cwd: __dirname, bundler: 'webpack' });
+		await api.export({ cwd: __dirname, bundler: 'webpack' });
+	});
+
+	it('injects <link rel=preload> tags', () => {
+		const index = fs.readFileSync(`${__dirname}/__sapper__/export/index.html`, 'utf8');
+		assert.ok(/rel=preload/, index);
```

Comment:
There is probably more I could do to test this, but I wasn't sure if you wanted me to include an entire HTML parsing library or something. (Cheerio?)